open-company/open-company-web
# Public post footer

Card: https://trello.com/c/AcQT2NG5

To test:
- copy a public URL
- open it in incognito
- [ ] do you see Learn more about Carrot at the bottom?
- [ ] if the post is really short do you still see it at the bottom of the page?
- click it
- [ ] did it go to homepage?
